auth-web/src/views/auth/home/IncorpOrRegisterView.vue

```diff
@@ -81,6 +75,15 @@ export default class IncorpOrRegisterView extends Vue {
   @Prop()
   private userProfile: User
 
+  private emitRedirectManage () {
+    if (this.userProfile) {
+      this.emitManageBusinesses()
+    } else {
+      // this is a redirect to the login page
+      window.location.href = this.dashboardUrl
```

Comment:
Maybe we need to append the account id here. So they will stay on the current account.
Can use `appendAccountId` function

Reply:
Or is there a `navigate()` global function (plugin) like the other UIs?

Reply:
Yes. I am changing it to `window.location.assign(appendAccountId(...`.

Reply:
Faiz,  can this conversation be resolved?